debian/changelog

```diff
@@ -1,3 +1,10 @@
+spdk (22.01.1-3) unstable; urgency=medium
+
+  [ Tom Nabarro ]
+  * Remove unnecessary --with-rdma configure option.
```

Comment:
You are not actually removing this flag in the Debian build: https://github.com/daos-stack/spdk/blob/1f940b949bca5a115411f01dd7fd25a18013ec34/debian/rules#L35

You will need to remove the line above also.

Reply:
done